README.md

```diff
@@ -114,7 +125,7 @@ See available metrics for your system
 $ $SNAP_PATH/bin/snapctl metric list
 ```
 
-Create task manifest file to use snap-plugin-collector-neutron plugin (exemplary file in examples/tasks/task.json):
+Create a task manifest file to use snap-plugin-collector-cinder plugin (exemplary file in [examples/tasks/] (https://github.com/intelsdi-x/snap-plugin-collector-neutron/blob/master/examples/tasks/)):
```

Comment:
Corrected in [pull request](https://github.com/intelsdi-x/snap-plugin-collector-neutron/pull/5)